tax_problem.rb

```diff
@@ -0,0 +1,45 @@
+require_relative "tax_helper.rb"
+include TaxHelper
+
+class Billing
+
+  def initialize(list_of_purchase)
+    @purchase_list = list_of_purchase
+  end
+
+  SALES_TAX = 5
+  BASIC_TAX = 10
+    
+  @purchase_list
+
+  def calculate_tax
+    for item in @purchase_list
```

Comment:
Avoid using `for` loop

Reply:
Changed to " @purchase_list.each do |item|
                       end